prov/rxd/src/rxd_attr.c

```diff
@@ -40,6 +40,7 @@
 
 struct fi_tx_attr rxd_tx_attr = {
 	.caps = RXD_EP_CAPS | RXD_TX_CAPS,
+	.op_flags = ~0ULL & ~FI_MULTICAST,
```

Comment:
I think the op_flags should be set in a way similar to the changes in rxm in #5027

Reply:
Updated